frontend/js/questions.js

```diff
@@ -1,53 +1,64 @@
+import APICall from "../common/APICall.js";
+
 const optionButtons = document.querySelectorAll('options-btn');
-//const modeToggleBtn = document.getElementById('mode-toggle-btn');
 const promptQuestion = document.getElementById('prompt-quetions');
 const progressBar = document.getElementById('progress-bar');
 const questionsCount = document.getElementById('current-question');
 const submitBtn = document.getElementById('submit')
 const BUTTONS_IDs = [];
-let correctAnswers = [];
+let SCORE=0;

```

Comment:
usually uppercased variable names mean constants and this ones are not constants

Reply:
Resolved